spring-cloud-stream-binder-kinesis-core/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -75,6 +96,10 @@ public void setCheckpoint(Checkpoint checkpoint) {
 
 		private long writeCapacity = 1L;
 
+		private int createTableDelay = 1;
+
+		private int createTableRetries = 25;
```

Comment:
Since both of them are definitely about table I would leave them without that word: `createDelay`, `createRetries`